src/Microsoft.ApplicationInsights.AspNetCore/TelemetryInitializers/OperationNameTelemetryInitializer.cs

```diff
@@ -15,7 +15,7 @@
 
     public class OperationNameTelemetryInitializer : TelemetryInitializerBase
     {
-        public const string BeforeActionNotificationName = "Microsoft.AspNet.Mvc.BeforeAction";
+        public const string BeforeActionNotificationName = "Microsoft.AspNetCore.Mvc.BeforeAction";
```

Comment:
@davidfowl @Eilon We noticed that the before action call back is not being invoked. Please let me know if there is any change in this support.